src/posteffect/posteffect_posteffectqueue.js

```diff
@@ -33,14 +33,18 @@ pc.extend(pc, function () {
          * @param {Boolean} useDepth Set to true if you want to create a render target with a depth buffer
          * @returns {pc.RenderTarget} The render target
          */
-        _createOffscreenTarget: function (useDepth) {
+
+        _createOffscreenTarget: function (useDepth, hdr) {
```

Comment:
Also question, do people use different formats here? instead of boolean, should this be a format?